cmake/CheckAlloca.cmake

```diff
@@ -0,0 +1,21 @@
+if(__check_alloca)
```

Comment:
Please include the license header in all source files.

Reply:
OK.